### PR TITLE
Document proxy.config.ssl.client.cipher_suite

### PR DESCRIPTION
**Pull request by persiaAziz**

### CODE REVIEW

File: doc/admin-guide/files/records.config.en.rst [hunk 1]
```diff
@@ -3132,6 +3132,10 @@ SSL Termination
 
    ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:DHE-RSA-AES256-GCM-SHA384:DHE-DSS-AES256-GCM-SHA384:DHE-RSA-AES128-GCM-SHA256:DHE-DSS-AES128-GCM-SHA256:ECDHE-ECDSA-AES256-SHA384:ECDHE-RSA-AES256-SHA384:ECDHE-ECDSA-AES256-SHA:ECDHE-RSA-AES256-SHA:ECDHE-ECDSA-AES128-SHA256:ECDHE-RSA-AES128-SHA256:ECDHE-ECDSA-AES128-SHA:ECDHE-RSA-AES128-SHA:DHE-RSA-AES256-SHA256:DHE-DSS-AES256-SHA256:DHE-RSA-AES128-SHA256:DHE-DSS-AES128-SHA256:DHE-RSA-AES256-SHA:DHE-DSS-AES256-SHA:DHE-RSA-AES128-SHA:DHE-DSS-AES128-SHA:AES256-GCM-SHA384:AES128-GCM-SHA256:AES256-SHA256:AES128-SHA256:AES256-SHA:AES128-SHA:DES-CBC3-SHA:!aNULL:!eNULL:!EXPORT:!DES:!RC4:!MD5:!PSK:!aECDH:!EDH-DSS-DES-CBC3-SHA:!EDH-RSA-DES-CBC3-SHA:!KRB5-DES-CBC3-SHA
 
+.. ts:cv:: CONFIG proxy.config.ssl.client.cipher_suite STRING NULL
+
+   Configures the cipher_suite which |TS| will use for SSL connections to origin or next hop.
```
Comment (SolidWallOfCode) [inline doc/admin-guide/files/records.config.en.rst]: "See notes under :ts:cv:`proxy.config.ssl.server.cipher_suite`."

Comment (persiaAziz) [reply doc/admin-guide/files/records.config.en.rst]: ?

Comment (persiaAziz) [reply doc/admin-guide/files/records.config.en.rst]: ts:cv: probably does not work in that place

**Comment by zwoop** [comment]
Cheery-picked to 7.1.x

**Pull request merged**